new-relic-deployment-marker/README.md

```diff
@@ -1,26 +1,30 @@
 # New Relic Deployment Marker
 Mark new deployments in New Relic with this Bitbucket Pipe
 
-## YAML Definition
-
-Add the following snippet to the script section of your `bitbucket-pipelines.yml` file:
-
-```yaml
-- pipe: docker://sykescottages/bitbucket-pipes:new-relic-deployment-marker
-    NEW_RELIC_API_KEY: '<string>'
-    NEW_RELIC_APPLICATION_ID: '<string>'
-    NEW_RELIC_REGION: '<string>'
-    DEPLOYMENT_USER: '<string>'
-    DEPLOYMENT_REVISION: '<string>'
-```
+## Important
+If you do not follow the new relic naming convention the deployment market might not work. Make sure you're following the naming convention when generating an APM.
```

Comment:
"the deployment market might not work" should be "the deployment marker might not work"